projects/admin/src/app/circulation/patron/profile/profile.component.html

```diff
@@ -15,6 +15,10 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <article *ngIf="currentPatron$ | async as patron">
+  <div *ngIf="canUpdate()" class="float-right">
+    <button [routerLink]="['/records','patrons','edit', patron.pid]" class="btn btn-sm btn-primary ml-1 ng-star-inserted" id="profile-edit-button" title="Edit"><i class="fa fa-pen mr-1"></i>{{ 'Edit' | translate }}</button>
+  </div>
+  <h3 class="mb-3">{{ patron.last_name }} {{ patron.first_name }}</h3>
```

Comment:
Optional case management for first name.

Reply:
Yes but for the moment the first_name is a required property. A lot of changes will be needed if we decide to make it optionnal.